cmake/onnxruntime_mlas.cmake

```diff
@@ -251,6 +265,49 @@ function(setup_mlas_source_for_windows)
   endif()
 endfunction()
 
+function(setup_kleidiai)
+  set(onnxruntime_USE_KLEIDIAI ${ARM64} CACHE BOOL "")
+  target_compile_definitions(onnxruntime_mlas PRIVATE $<$<BOOL:${onnxruntime_USE_KLEIDIAI}>:USE_KLEIDIAI>)
+
+  if (onnxruntime_USE_KLEIDIAI)
+    # Disable the KleidiAI tests
+    set(KLEIDIAI_BUILD_TESTS  OFF)
+
+    # Fetch KleidiAI sources:
+    if (NOT TARGET kleidiai)
+      if (POLICY CMP0135)
```

Comment:
Do you need to set this policy?

This policy was introduced in CMake version 3.24. We require cmake version >=3.28. So the policy is already NEW.